src/features/interview/select-interviewer-box.tsx

```
@@ -22,7 +22,9 @@ const SelectInterviewerBox = () => {
   const { resumeId } = useResumeStore();
   const resetQuestionIndex = useInterviewStore((state) => state.resetQuestionIndex);
 
-  {/** TODO: clay이냐 poly 타입이냐에 따라 수정해야됨 */}
+  {
+    /** TODO: clay이냐 poly 타입이냐에 따라 수정해야됨 */
+  }
   const charterType = 'poly';
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)